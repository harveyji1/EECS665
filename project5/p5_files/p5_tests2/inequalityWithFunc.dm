fn : () void {}

main : () void {
    a : bool;
	a = (fn() == 5);
}
