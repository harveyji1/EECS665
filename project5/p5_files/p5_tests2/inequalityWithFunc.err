FATAL [5,7]-[5,16]: Invalid equality operation
Type Analysis Failed
