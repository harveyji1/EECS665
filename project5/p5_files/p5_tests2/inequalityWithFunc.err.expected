FATAL [5,7]-[5,11]: Invalid equality operand
Type Analysis Failed
